aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -176,41 +170,6 @@ impl DeltaOp {
         self.merge_with_previous_delta(previous_delta)?;
         Ok(())
     }
-
-    /// Consumes a single delta and tries to materialize it with a given state
-    /// key. If materialization succeeds, a write op is produced. Otherwise, an
-    /// error VM status is returned.
-    pub fn try_into_write_op(
```

Comment:
Why is `try_into_write_op` removed? It's being used for materializing deltas right?

Reply:
Had to move to resolver :(